src/Integration.UnitTests/Service/SonarQubeServiceWrapperTests.cs

```diff
@@ -155,6 +155,47 @@ public void SonarQubeServiceWrapper_TryGetProjects_Organization()
         }
 
         [TestMethod]
+        public void SonarQubeServiceWrapper_TryGetNotifications()
+        {
+            var expected = new NotificationEvent[]
+            {
+                new NotificationEvent { Category = "QUALITY_GATE",
+                    Message =  "Quality Gate of project 'test' is now Red (was Green)",
+                    Link = new Uri("http://localhost:9000/dashboard?id=test"),
+                    Date = new DateTimeOffset(2017, 1, 1, 9, 55, 1, 0, TimeSpan.FromHours(1)),
+                    Project = "test" }
+            };
+
+            using (var testSubject = new TestableSonarQubeServiceWrapper(this.serviceProvider))
+            {
+                testSubject.RegisterQueryValidator("api/developers/search_events", request =>
+                {
+                    request.QueryString.HasValue.Should().BeTrue();
```

Comment:
Is it sanity check (if so please add a comment) otherwise, isn't it too early for the assertion (as you are following AAA pattern).

Reply:
I've borrowed the pattern from other test method. Do you have a suggestion on how to change it? Is the problem likely to go away after your client rewrite?

Reply:
This has changed yes!